Rocket.Chat/Managers/Model/SubscriptionManager.swift

```diff
@@ -46,6 +46,11 @@ struct SubscriptionManager {
             SocketManager.send(requestRooms) { response in
                 guard !response.isError() else { return Log.debug(response.result.string) }
 
+                guard let auth = AuthManager.isAuthenticated() else { return }
```

Comment:
@cardoso @rafaelks 
When adding a new server this results in
```
libc++abi.dylib: terminating with uncaught exception of type NSException
(lldb) 
```
So App crashes everytime adding a new server! 😢